plasma_cash/child_chain/server.py

```diff
@@ -61,3 +61,9 @@ def relay(ws, arg):
     dest = clients[arg['dest']]
     msg = arg['message']
     dest.send(json.dumps({'event': 'relay', 'arg': msg}))
+
+
+@on('block')
+def on_block(block_number):
+    for ws in clients.values():
+        ws.send(json.dumps({'event': 'block', 'arg': block_number}))
```

Comment:
why send block_number instead of the whole block data?

Reply:
clients don't need to know the whole block data, and whole block data is considered against the privacy benefits from plasma cash. However, clients do need a signal at new block creation for like monitoring their own asset (by requesting non-inclusion proof).